gsy_framework/community_datasheet/row_converters.py

```diff
@@ -0,0 +1,88 @@
+import uuid
+from datetime import datetime
+from typing import Dict
+
+import pendulum
+
+from gsy_framework.community_datasheet.sheet_headers import (
+    LoadSheetHeader, PVSheetHeader, StorageSheetHeader)
+from gsy_framework.constants_limits import DATE_TIME_FORMAT, ConstSettings
+
+
+class GeneralSettingsRowConverter:
+    """Convert from the excel row to a grid representation of a Load asset."""
+
+    @classmethod
+    def convert(cls, row: Dict) -> Dict:
+        """Convert the row using just the field name and value.
+
+        The field name is converted to snake case.
+        """
+        field_name, value = row[0], row[2]  # Each row is a tuple (setting-name, legend, value)
+        field_name = field_name.strip().lower().replace(" ", "_")
+        if isinstance(value, datetime):
+            value = pendulum.instance(value).format(DATE_TIME_FORMAT)
+
+        return {field_name.strip().lower(): value}
+
+
+class LoadRowConverter:
+    """Convert from the excel row to a grid representation of a Load asset."""
+
+    @classmethod
+    def convert(cls, row: Dict) -> Dict:
+        """
+        Convert the row parsed from the Excel file to the asset representation used in scenarios.
+        """
+        # One of the following:
+        # ["LoadHoursStrategy", "DefinedLoadStrategy",
+        # "LoadHoursExternalStrategy", "LoadProfileExternalStrategy",
+        # "LoadForecastExternalStrategy", "Load"]
```

Comment:
> Gsy-web already does this mapping (in DBToSimulationHelpers). To avoid duplicating code, I'd like to refactor that part and re-use it while importing the datasheet grid into gsy-web. This is not so straightforward though, because there are many operations performed by that class. That's what I'd like to work on today.

Now that I know that only profiles are allowed for the energy setup of Loads and PVs, I do not think your need to do that any more. Also because it was decided to not support direct use of the datasheet on gsy-e. The gsy-webs 'DBToSimulationHelpers' will take care of it right ? (I am aware of the fact that this class might need some refactorisation, but nothing to do now IMO).

Reply:
I'll double-check soon. I had this plan before encountering the hiccups with `CloudCoverage`, but you might be right saying that this might not be needed anymore!